ui/page.go

```diff
@@ -680,6 +704,10 @@ func (page pageCommon) SelectedAccountLayout(gtx layout.Context) layout.Dimensio
 	return card.Layout(gtx, selectedDetails)
 }
 
+func (page pageCommon) UniformPadding(gtx layout.Context, body layout.Widget) layout.Dimensions {
+	return layout.UniformInset(values.TextSize16).Layout(gtx, body)
```

Comment:
Using a dimension defined as TextSize for padding or margin doesn't look natural. You should define the dimension as MarginPadding. Text sizes should be defined as sp units anyway rather than dp.